### PR TITLE
Docker file added

### PR DESCRIPTION
**Pull request by alirezamdv**
Hi, i added Dockerfile and docker-compose.yml and test it everything is fine...i also edited README

### CODE REVIEW

File: dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+ENV PYTHONUNBUFFERED 1
+
+RUN mkdir /parking_manager
+
+WORKDIR /parking_manager
+
+ADD . /parking_manager/
+
+RUN pip install -r requirements.txt
```
Comment (gdalmau) [inline dockerfile]: Since it's a library that can be installed isn't it better to `pip install .` ?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+version: '3'
+
+services:
+  web:
+    build: .
+    command: bash -c "cd parking_manager && python manage.py makemigrations && python manage.py migrate && python manage.py loaddata parking_app/fixtures/*.json && python manage.py runserver 0.0.0.0:8000"
```
Comment (gdalmau) [inline docker-compose.yml:6]: I was thinking in placing all the commands into the `dockerfile` and in the `docker-compose.yml` just use it like this:

```yml
services:
  python:
    build:
      dockerfile: Dockerfile
```

What do you think?

Comment (alirezamdv) [reply docker-compose.yml:6]: hi, I would say we can do it in many ways ... I learned that from the documentation and I think it's just fine..

File: dockerfile (hunk 1 above)
Comment (alirezamdv) [reply dockerfile]: of course! I changed it

**Pull request merged**